jung-visualization/src/main/java/edu/uci/ics/jung/visualization/transform/LayoutLensSupport.java

```diff
@@ -87,6 +90,7 @@ public void deactivate() {
     }
     vv.setToolTipText(defaultToolTipText);
     vv.setGraphMouse(graphMouse);
+    vv.setTransformSupport(new TransformSupport<>());
     vv.repaint();
   }
 }
```

Comment:
All of the *LensSupport classes now also set the transformSupport in vv depending on whether the Lens effects are activated or deactivated